manifester/manifester.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, manifest_category, allocation_name=None, **kwargs):
 
             self.requester = requests
             self.is_mock = False
-        self.allocation_name = allocation_name or "".join(random.sample(string.ascii_letters, 10))
+        self.allocation_name = allocation_name or f"{settings.username_prefix}-" + "".join(
+            random.sample(string.ascii_letters, 10)
+        )
```

Comment:
Haven't tested this but I think this would work:
```suggestion
        self.allocation_name = allocation_name or f"{settings.username_prefix}-".join(
            random.sample(string.ascii_letters, 10)
        )
```

Reply:
Your suggestion was the first thing I tried, but I found that it joined the f-string separately to each character in the random ASCII string, e.g. `tusername-Musername-gusername-Ousername-i`.

Reply:
Yikes...